src/python/WMCore/WorkQueue/DataStructs/WorkQueueElementResult.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, **kwargs):
                             sum([x['EventsWritten'] for x in self['Elements']]))
             self.setdefault('FilesProcessed',
                             sum([x['FilesProcessed'] for x in self['Elements']]))
+            self.setdefault('Jobs',
```

Comment:
Don't we need to consider the blowupFactor here?
And a silly question, what's the difference in this if-else? Is the ParentQueueElement the workqueue_inbox doc that is created for each workflow, and which spawns > 1 WQE?